src/playbooks/services/services.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Manage services
+  hosts: quadlet
+  gather_facts: true
+  become: true
+  vars_files:
+    - "../../vars/defaults.yml"
+    - "../../vars/database.yml"
+  tasks:
+    - name: "Manage services"
```

Comment:
I wonder if we should create a systemd target that groups all of the services together for this operation of controlling all of them being started and stopped.